### PR TITLE
test(controllers): Added basic controllers + controller tests

### PR DESCRIPTION
**Pull request by peterdolan**
What: FINALLY have the correct includes here for controllers. I learned a ton about git rebasing during this process, but this should be correct. Basically has controller (and only controller) updates here, as well as a couple minute changes to the files that were already here.
Why: To get this commit the way it was supposed to be!

### CODE REVIEW

File: lib/server.js [hunk 1]
```diff
@@ -18,9 +18,9 @@ server.register([
     register: require('hapi-format-error'),
     options: { joiStatusCode: 422 }
   },
-  require('./plugins/features/movies')
+  require('../lib/plugins/features/movies')
```
Comment (charlesbjohnson) [inline lib/server.js]: I think this can be removed.

Comment (peterdolan) [inline lib/server.js]: done

**Comment by charlesbjohnson** [comment]
LGTM

File: test/plugins/features/movies/index.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      .then((response) => {
+        expect(response.statusCode).to.eql(200);
+        expect(response.result.object).to.eql('movie');
+      });
+    });
+
+  });
+
+});
+
```
Comment (robinjoseph08) [inline test/plugins/features/movies/index.test.js]: nit: extra line

**Comment by robinjoseph08** [comment]
one thing but LGTM!

**Pull request merged**